packages/graphback/src/resolvers/knex/resolverImplementation.ts

```diff
@@ -1,17 +1,22 @@
 const knexContext = `context.db`
-const argumentContext = `args`
 
-export const createTemplate = (fieldName: string, tableName: string): string => {
+export const createTemplate = (fieldName: string, tableName: string, typeName: string): string => {
     return `${fieldName}: async (root, args, context, info) => {
-      const result = await ${knexContext}('${tableName}').insert(${argumentContext}.input).returning('*')
+      const result = await ${knexContext}('${tableName}').insert(args.input).returning('*')
+      pubsub.publish(Subscriptions.NEW_${typeName.toUpperCase()}, {
```

Comment:
pubsub must be on context. Not sure where this comes from

Reply:
Done